templates/html_evidence.fields

```diff
@@ -0,0 +1,13 @@
+issue.host
```

Comment:
shouldn't all of these be `html_evidence.` instead of `issue.`?

Reply:
I am not sure, probably yes.
We are parsing a fragment of data that can contain both issue/evidence info.
So in some way it also makes sense to keep using issue, as we do in `evidence.fields` file.